assets/src/settings-page/settings-footer.js

```diff
@@ -2,49 +2,70 @@
  * WordPress dependencies
  */
 import { Button } from '@wordpress/components';
-import { useContext, useEffect } from '@wordpress/element';
+import { useContext, useEffect, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { Options } from '../components/options-context-provider';
 import { ReaderThemes } from '../components/reader-themes-context-provider';
+import { AMPNotice, NOTICE_TYPE_SUCCESS } from '../components/amp-notice';
+import { ErrorContext } from '../components/error-boundary';
 
 /**
  * The bottom section of the settings page.
  */
 export function SettingsFooter() {
-	const { didSaveOptions, editedOptions, saveOptions, savingOptions } = useContext( Options );
+	const error = useContext( ErrorContext );
+	const { didSaveOptions, editedOptions, hasOptionsChanges, saveOptions, savingOptions } = useContext( Options );
 	const { downloadingTheme } = useContext( ReaderThemes );
+	const [ saved, setSaved ] = useState( false );
 
 	const { reader_theme: readerTheme, theme_support: themeSupport } = editedOptions;
 
 	/**
-	 * Submits the PHP-generated form on the page after options have saved via REST.
-	 *
-	 * @todo Eliminate this once the Plugin Suppression logic is moved to use the REST API.
+	 * Shows a saved notice on success.
 	 */
 	useEffect( () => {
 		if ( true === didSaveOptions && ! downloadingTheme ) {
-			document.querySelector( 'form#amp-settings' ).submit();
+			setSaved( true );
+
+			const timeout = setTimeout( () => [
+				setSaved( false ),
+			], 9000 );
+
+			return () => {
+				clearTimeout( timeout );
+			};
 		}
+
+		return () => undefined;
 	}, [ didSaveOptions, downloadingTheme ] );
 
 	const disabled = ! themeSupport ||
+		! hasOptionsChanges ||
+		error ||
```

Comment:
As noted in https://github.com/ampproject/amp-wp/pull/5044#issuecomment-659015797 I think the `error` condition needs to be removed here, or modified in some way so that if an error happened during saving the user can try saving again.

Reply:
@westonruter I have the approach worked out and will be working through it.

Reply:
I think 08e83151c4e96ddd28491779d899ad8fb601b766 will do what we need, but I need to double-check it this evening.

Reply:
Yes, that seems to work well!